assets/js/atomic/utils/get-block-map.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { getRegisteredInnerBlocks } from '@woocommerce/blocks-registry';
+
+/**
+ * Internal dependencies
+ */
+import ProductButton from '../blocks/product/button/frontend';
```

Comment:
Rather than using atomic/components, this is mapping to the frontend.js file of each defined Atomic Block. The frontend.js file is responsible for converting dataset attributes to block attributes, then rendering the block's component.